self-core-impl/src/main/java/com/selfxdsd/core/StoredUser.java

```diff
@@ -102,6 +112,24 @@ public Projects projects() {
         return this.storage.projects().ownedBy(this);
     }
 
+    @Override
+    public Organizations organizations() {
+        final Organizations organizations;
+        switch (this.provider){
+            case Provider.Names.GITHUB: organizations =
+                new GithubOrganizations(this.resources,
+                    URI.create("https://api.github.com"), this,
+                    this.storage);
+                break;
```

Comment:
@amihaiemil Having Provider.Names as enum that encapsulates the name and provider Uri would be really handy :).
If not just class of constants (eg: Provider.Uri) would do it too. Right now Github uri is copy/pasted in a bunch of places.

Reply:
@criske I see, but let's leave it as it is for now. We'll make these kind of small imrovements later :D